### PR TITLE
Moving to React Native 0.60.4

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

File: ReactNativeTemplate/android/app/build.gradle [hunk 1]
```diff
@@ -86,26 +86,36 @@ def enableSeparateBuildPerCPUArchitecture = false
...
+ */
+def useIntlJsc = false
+
 android {
-    compileSdkVersion 28
-    buildToolsVersion '28.0.3'
+    compileSdkVersion rootProject.ext.compileSdkVersion
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
```
Comment (bhariharan) [inline ReactNativeTemplate/android/app/build.gradle]: This is likely to fail on some `API 21` and `API 22` devices - they should be capable of running `Java 1.6` if I'm not mistaken.

File: ReactNativeTemplate/android/build.gradle [hunk 2]
```diff
@@ -1,6 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+  ext {
+    buildToolsVersion = "28.0.3"
+    minSdkVersion = 21
+    compileSdkVersion = 28
+    targetSdkVersion = 28
+    supportLibVersion = "28.0.0"
```
Comment (bhariharan) [inline ReactNativeTemplate/android/build.gradle]: Shouldn't this be the same as `buildToolsVersion`?

File: SmartSyncExplorerReactNative/android/app/build.gradle [hunk 3]
```diff
@@ -87,26 +87,36 @@ def enableSeparateBuildPerCPUArchitecture = false
...
+ */
+def useIntlJsc = false
+
 android {
-    compileSdkVersion 28
-    buildToolsVersion '28.0.3'
+    compileSdkVersion rootProject.ext.compileSdkVersion
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
```
Comment (bhariharan) [inline SmartSyncExplorerReactNative/android/app/build.gradle]: Same comment as above. Do we need this?

File: ReactNativeTemplate/android/app/build.gradle (hunk 1 above)
Comment (wmathurin) [reply ReactNativeTemplate/android/app/build.gradle]: I ran their CLI to generate an application and copied over their build.gradle.

File: ReactNativeTemplate/android/build.gradle (hunk 2 above)
Comment (wmathurin) [reply ReactNativeTemplate/android/build.gradle]: I don't think there is a 28.0.3 version of the support lib (see https://developer.android.com/topic/libraries/support-library/revisions).

**Comment by bhariharan** [comment]
Cool. LGTM!

**Pull request merged**